classes/class-aal-activity-log-list-table.php

```diff
@@ -443,24 +439,25 @@ public function prepare_items() {
 		$offset = ( $this->get_pagenum() - 1 ) * $items_per_page;
 
 		
-		$total_items = $wpdb->get_var( $wpdb->prepare(
-			'SELECT COUNT(`histid`) FROM `%1$s`
+		$total_items = $wpdb->get_var( 
+			'SELECT COUNT(`histid`) FROM `' . $wpdb->activity_log . '`
 				' . $where . '
-					' . $this->_get_where_by_role(),
-			$wpdb->activity_log,
-			$offset,
-			$items_per_page
-		) );
+					' . $this->_get_where_by_role() 
+		);
 		
+		
+		$items_orderby = filter_input(INPUT_GET, 'orderby', FILTER_SANITIZE_STRING);
```

Comment:
'orderby' is empty by default, so we should set a fallback here, otherwise, no results will be loaded:

`if (empty($items_orderby)) {
    $items_orderby = 'hist_time';
}`